README.md

```diff
@@ -1,2 +1,6 @@
 # gae-java
 Support for Spine-based Java apps running under Google App Engine.
+
+GAE Datastore startup script is named `start-datastore.sh`. On windows, make the same `.bat` file or
```

Comment:
Actually, the steps are the same, but the script contains necessary parameters, such as project name. I think I'll just move this info from README.md to wiki.